internal/worker.go

```diff
@@ -205,6 +205,10 @@ type (
 		// Optional: Sets opentracing Tracer that is to be used to emit tracing information
 		// default: no tracer - opentracing.NoopTracer
 		Tracer opentracing.Tracer
+
+		// Optional: Automatically send heartbeats for any activity with a HeartbeatTimeout
```

Comment:
Also we should state it clear here that this cannot be used with calling heartbeat in the code with details. It will lead to wrong behavior(losing the details progerss)